htdocs/fourn/class/fournisseur.commande.class.php

```diff
@@ -1573,7 +1574,7 @@ public function create($user, $notrigger = 0)
 		$sql .= ", ".setEntity($this);
 		$sql .= ", ".((int) $this->socid);
 		$sql .= ", ".($this->fk_project > 0 ? ((int) $this->fk_project) : "null");
-		$sql .= ", '".$this->db->idate($date)."'";
+		$sql .= ", '".$this->db->idate($now)."'";
```

Comment:
This means we can't create anymore an order with the date of our choice.
The use of $now was already managed on line 1522
```
if (empty($date)) {
			$date = $now;
		}
```
So to have the current date when cloning we must set ->date = dol_now() into the clone method.

Reply:
Thank you for you feedback.
I fix this now. :)